### DISCUSSION TITLE
Trouble deploying to fly.io

### DISCUSSION THREAD

**Discussion by iqfareez** [flet-dev/flet#1316] [Q&A]
### Question

I'm following the docs to deploy the flet app.
https://flet.dev/docs/guides/python/deploying-web-app/hosting-providers/fly-io

However, during deployment, there is an error with the fly.io builder:

```
[error] Health check on port 8080 has failed. Your app is not responding properly. Services exposed on ports [80, 443] will have intermittent failures until the health check passes.
```

![image](https://user-images.githubusercontent.com/60868965/233508983-ae7c1c77-7a3a-4a90-b2bd-767be99e6ac0.png)

Anyone can help or can provide some clue to this issue?

### Code sample

```python
NA
```

### Error message

```bash
[error] Health check on port 8080 has failed. Your app is not responding properly. Services exposed on ports [80, 443] will have intermittent failures until the health check passes.
```

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by bm7at** [comment]
I got the same issue, did you find a solution for this? :/

**Reply by ndonkoHenri** [reply]
Can you try with a very simple app (counter?) and share the results?
Remember to set the `view=ft.WEB_BROWSER`.

**Reply by bm7at** [reply]
@ndonkoHenri Have added view=ft.WEB_BROWSER after, it is working now. :) Thanks much, very appreciate it.

Do you still work with flet?

I have seen your repos, learned a lot from it. 👍

**Reply by ndonkoHenri** [reply]
Happy It helped.

> Do you still work with flet?

Yes I do.
😄

**Comment by FeodorFitsner** [comment]
Pay attention to `FLET_FORCE_WEB_VIEW = "true"` environment variable that must be added to `fly.toml`.

**Reply by ndonkoHenri** [reply]
Should this env variable always be set when deploying as Web app, or it's just related to fly.io ?

And also, what is it concrete role? I mean in contrast to `view=web`.

**Reply by FeodorFitsner** [reply]
It's either `view=ft.AppView.WEB_BROWSER` or `FLET_FORCE_WEB_VIEW = "true"`.
